database/dbPersons.php

```diff
@@ -182,6 +182,67 @@ function update_volunteer_hours($eventname, $username, $new_start_time, $new_end
 	return $result;
 }
 
+/*@@@ Thomas */
+
+/* Check-in a user by adding a new row and with start_time to dbpersonhours */
+function check_in($personID, $eventID, $start_time) {
+    $con=connect();
+	$query = "INSERT INTO dbpersonhours (personID, eventID, start_time) VALUES ( '" .$personID. "', '" .$eventID. "', '" .$start_time. "')";
+    //$query = 'UPDATE dbpersonhours SET start_time = "' . $new_start_time . '", end_time = "' . $new_end_time . ' WHERE eventID = "' . $eventid . '" AND personID = "' . $username . '"';
```

Comment:
what does this query commented out do, and why is it commented out?

Reply:
I was using that as a reference for writing the query above. At this point it's useless so I'll remove it.